packages/components/src/TextField/TextField.module.scss

```diff
@@ -0,0 +1,49 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "../../styles/animation";
+@import "../../styles/legacy/layout";
+@import "../../styles/legacy/type";
+
+@mixin input-icon($color, $reversed) {
+  color: $color;
+
+  .withReversed & {
+    color: $reversed;
+  }
+}
+
+.input {
+  margin-top: $spacing-6;
+}
+
+$input-disabled-opacity: 0.3;
+
+///////////////////////////////////////////////////
+// VALIDATION STYLES
+///////////////////////////////////////////////////
+.success {
+  @include input-icon($color-green-500, $color-green-400);
```

Comment:
Noting for future: would be good to consolidate these into one place for all of the inputs

Reply:
You actually pointed out an important thing, TextAreaField doesn't have a `success` validation state lol